src/components/search/SearchProgramCard.jsx

```diff
@@ -79,7 +79,7 @@ const SearchProgramCard = ({ hit, isLoading, ...rest }) => {
   };
 
   const primaryPartnerLogo = getPrimaryPartnerLogo(partnerDetails);
-
+  const defaultImage = 'https://prod-discovery.edx-cdn.org/media/programs/card_images/2f69f430-7a9a-4100-b349-40d4787c0af2-494a6fcbed02.jpg';
```

Comment:
IF we are going to use a hardcoded image URL like this, we should do so via a configuration setting (e.g., `.env.development`).

That said, I would recommend using the fallback image intended to be used by the `Card` when no image exists but one should be rendered. 

See the `HighlightedContentCard` as an example.

https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/8e6f70d05ff6d8d0263e72c746821cce9ad62da7/src/components/search/content-highlights/HighlightedContentCard.jsx#L63-L69

I'm wondering if we could do something like the following?

```jsx
import cardImageCapFallbackSrc from '@edx/brand/paragon/images/card-imagecap-fallback.png';

<Card.ImageCap
  src={cardImageUrl || cardImageCapFallbackSrc}
  fallbackSrc={cardImageCapFallbackSrc}
  srcAlt=""
  // ...
/>
```